portfolio/src/main/webapp/script.js

```diff
@@ -16,21 +16,31 @@
  * Adds a random greeting to the page.
  */
 
- import "nes.css/css/nes.min.css";
-
 function randMessage() {
     const msgList =
         ['“With a successful attack roll, the wizard maintains the thief in surfboard position.”',
-         'DM: So, what are your character’s flaws? Player: Acid reflux.',
-          'That’s too indiscriminate.  We should go with the rampaging bear.',
-          'We’ve managed to teach the robot about the most important human emotion: Humiliation.',
-          '“Do you have a permit for that facial hair?”',
-          'Cleric: “Everybody chill, it’s totes God’s will.”',
-          'Did asterisks exist in 2006?'];
+            'DM: So, what are your character’s flaws? Player: Acid reflux.',
+            'That’s too indiscriminate.  We should go with the rampaging bear.',
+            'We’ve managed to teach the robot about the most important human emotion: Humiliation.',
+            '“Do you have a permit for that facial hair?”',
+            'Cleric: “Everybody chill, it’s totes God’s will.”',
+            'Did asterisks exist in 2006?'];
     const msg = msgList[Math.floor(Math.random() * msgList.length)];
     const msgContainer = document.getElementById('message-container');
     msgContainer.innerText = msg;
 }
+function servlet() {
+    console.log("servlet has been called")
+    fetch('/data').then(response => response.json())//fetch from data
+        .then((someTexts) => {
+            console.log(someTexts);
+            document.getElementById('quote-container').innerText = someTexts;
+
+            const hist = document.getElementById('quote-container');
+            someTexts.forEach((line)=> {hist.appendChild(createListElement(line))}
```

Comment:
here it looks like "someTexts" will be written twice in the 'quote-container' element. One as just a json array, the other as a html list. Also, you make call to the "createListElement" method, but I don't see that method here? The reason that you only see the comments once in your page might be that this someTexts.forEach((line)=> {hist.appendChild(createListElement(line))}) code doesn't compile (because there is no createListElement method)

Reply:
That would make a lot of sense if it wasn't compiling. The latter section was supposed to put each comment on a new line, so that will be the one I'd like to keep. I saw the createListElement used in other files though, what can I do to implement it properly?

Reply:
Thanks for catching that! This will be fixed in the next commit!